packages/origin-ui-core/src/components/Account/UserProfile/Styles.tsx

```diff
@@ -0,0 +1,19 @@
+import { createStyles, makeStyles, useTheme } from '@material-ui/core';
+
+const useStyles = makeStyles(() =>
+    createStyles({
+        container: {
+            padding: '20px'
+        },
+        buttonAndIconHolder: {
+            paddingTop: '1rem',
+            display: 'flex',
+            alignItems: 'center'
+        },
+        infoIcon: {
+            marginLeft: '1rem'
+        }
+    })
+);
+
+export const useUserProfileStyle = () => useStyles(useTheme());
```

Comment:
Why do you need UserProfile styles as a separate component? Why don't include it into the `UserDataEditForm.tsx` ?

Reply:
it's also used in every other part of user profile page (ChangePassword, UserEmailChange, etc). should I use different approach is case like this?

Reply:
Then you can create an scss file, like UserProfile.scss in styles folder and apply classes

Reply:
done, styles moved to scss file